add examples using scroll an animation libraries

P1X3L opened the pull request:
https://trello.com/c/4Dj5wiEd/3090-mentionner-avec-exemple-comment-utiliser-la-lib-react-custom-scrollbars-ou-les-csstransitiongroup-pour-la-liste-avec-la-prop-lis
Signed-off-by: Paul-Xavier Ceccaldi <pix@wttj.co>

P1X3L committed:
add examples using scroll an animation libraries

https://trello.com/c/4Dj5wiEd/3090-mentionner-avec-exemple-comment-utiliser-la-lib-react-custom-scrollbars-ou-les-csstransitiongroup-pour-la-liste-avec-la-prop-lis
Signed-off-by: Paul-Xavier Ceccaldi <pix@wttj.co>

klacointe commented on gh-pages/examples/CssTransitionExample.js:
```diff
@@ -0,0 +1,46 @@
+        classNames='fade'
+        unmountOnExit={true}
+      >
+        {list}
```
I think `list` should be called `children`

klacointe commented on gh-pages/examples/CustomScrollbarsExample.js:
```diff
@@ -0,0 +1,50 @@
+            height: 350
+          }}
+        >
+          {list}
```
Same here

klacointe commented on src/JungleSelect.js:
```diff
@@ -620,17 +620,21 @@ class JungleSelect extends Component {
+              className='jungle-select-list'
+              ref={(e) => this.itemsContainer = e }
+            >
+              {this.renderList()}
```
the `listWrapper` can be done in the `renderList` function to keep the render function light

P1X3L committed:
clean render function and js gardening

https://trello.com/c/4Dj5wiEd/3090-mentionner-avec-exemple-comment-utiliser-la-lib-react-custom-scrollbars-ou-les-csstransitiongroup-pour-la-liste-avec-la-prop-lis
Signed-off-by: Paul-Xavier Ceccaldi <pix@wttj.co>

klacointe commented:
@P1X3L toujours d'actualité ?

P1X3L commented:
@klacointe, je ferme c'est trop vieux !
